Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/BaseLinearElasticityFEMForceField.inl

```diff
@@ -28,14 +28,58 @@ namespace sofa::component::solidmechanics::fem::elastic
 
 template <class DataTypes>
 BaseLinearElasticityFEMForceField<DataTypes>::BaseLinearElasticityFEMForceField()
-    : d_poissonRatio(initData(&d_poissonRatio,(Real)0.45,"poissonRatio","FEM Poisson Ratio in Hooke's law [0,0.5["))
-    , d_youngModulus(initData(&d_youngModulus, defaultYoungModulusValue, "youngModulus","FEM Young's Modulus in Hooke's law"))
+    : d_poissonRatio(initData(&d_poissonRatio, { defaultPoissonRatioValue }, "poissonRatio", "FEM Poisson Ratio in Hooke's law [0,0.5["))
+    , d_youngModulus(initData(&d_youngModulus, { defaultYoungModulusValue }, "youngModulus", "FEM Young's Modulus in Hooke's law"))
     , l_topology(initLink("topology", "link to the topology container"))
 {
     d_poissonRatio.setRequired(true);
-    d_poissonRatio.setWidget("poissonRatio");
-
     d_youngModulus.setRequired(true);
+
+    this->addUpdateCallback("checkPoissonRatio", {&d_poissonRatio}, [this](const core::DataTracker& )
+    {
+        checkPoissonRatio();
+        return this->getComponentState();
```

Comment:
I think it is changed for the same value (but the counter will change and can trigger change on other object)

Reply:
@hugtalbot this has to be clarified. Let's say I have an invalid state component, for example because it has been initialized and an error appeared. For some reasons, the code accesses to `d_poissonRatio`. It triggers the callback. If the callback returns `ComponentState::valid`, it will validate the state of the component, but nothing has been really made to fix the error that appeared in the init. That is why I return the current state.

If you return `ComponentState::valid`, I think a unit test starts to fail because of the reason I mentioned.